examples/all-clusters-app/all-clusters-common/src/operational-state-delegates.cpp

```diff
@@ -0,0 +1,184 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#include <app-common/zap-generated/attributes/Accessors.h>
+#include <app/EventLogging.h>
+#include <app/clusters/operational-state-server/operational-state-server.h>
+#include <app/reporting/reporting.h>
+#include <app/util/config.h>
+#include <operational-state-delegate-impl.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace OperationalState {
+
+/**
+ * class to present Enquriy Table of Delegates
+ */
+struct DelegatesEnquiryTable
+{
+    /**
+     * Endpoint Id
+     */
+    EndpointId mEndpointId;
+    /**
+     * Cluster Id
+     */
+    ClusterId mClusterId;
+    /**
+     * point of Array(Items)
+     */
+    Delegate * pItems;
```

Comment:
I don't get it.  For a given endpoint+cluster there should be exactly one delegate.